src/server/collections/videos-db.js

```diff
@@ -47,5 +50,68 @@ function findVideoInfo(videoId, callback) {
   });
 }
 
-module.exports = {findVideoInfo: findVideoInfo};
+function addCommentBuilder(userInfo, videoId, content, callback) {
+  MongoClient.connect(url, (err, db) => {
+    try {
+      if (err) {
+        throw err;
+      }
+      if (videoId.length !== 24) {
+        return callback([]);
+      }
+      let videosDB = db.collection('videos');
+
+      videosDB.findOne({'_id': ObjectId(videoId)}, (err, videoInfo) => {
+        if (err) {
+          console.log(err);
+        }
+        let commentId = videoInfo
+          .commentInfos[videoInfo.commentInfos.length - 1].commentId + 1;
+        let obj = {
+          'username': userInfo.username,
+          'userId': userInfo._id,
+          'avatar': 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQLDxSdH8lLX-y9TJzLDWZPvoLexXrE8Ft5EAAWaZNyQHVM-yh-3A',
+          'commentTime': new Date().getTime(),
+          'likeNum': 0,
+          'dislikeNum': 0,
```

Comment:
we dont need these numbers, since we can caluculate on the fly when we are sending the data to the client